dali/operators/random/uniform_distribution_cpu.cc

```diff
@@ -30,10 +34,15 @@ generated.
     .AddOptionalArg("range",
       R"code(Range ``[min, max)`` of a continuous uniform distribution.
 
-This argument is mutually exclusive with ``values``.)code",
+This argument is mutually exclusive with ``values``.
+
+.. warning::
+  When specifying an integer type as ``dtype``, the generated numbers can go outside
+  the specified range, due to rounding.
```

Comment:
> the generated numbers can go outside the specified range

Can't we clamp? It is a bit unexpected.

Reply:
So the RNG work like that, you generate the number in whatever is the natural type for the distribution, in this case float. Then if the user request the output data type to be a different type, we just ConvertSat<Out>(...). It's equivalent as doing ```fn.cast( fn.random.uniform(...), dtype=...)```, the numbers when converted to an integer data type, can go outside of the float range that was specified for the distribution. If the user needs a continuous distribution, it doesn't make much sense to request dtype to be an integer data type, I'd say.

Reply:
Make sense.